core/src/assets/login.js

```diff
@@ -61,13 +79,38 @@ async function onKubeconfigUploaded(e) {
       });
     } else {
       toggleOIDCForm(true);
+      toggleConfigForm(false);
+      toggleBackButton(true);
     }
   } catch (e) {
     toggleError(true);
     console.warn(e);
   }
 }
 
+async function onKubeconfigPasted(kubeconfig, type) {
+  displayInitialView();
+  try {
+    const kk = jsyaml.load(kubeconfig);
```

Comment:
Even in this context, I think this name is too mysterious. What about `kubeconfigParsed`?